src/index/base.cpp

```diff
@@ -160,7 +160,7 @@ void BaseIndex::ThreadSync()
                 pindex = pindex_next;
             }
 
-            int64_t current_time = GetTime();
+            std::chrono::seconds current_time = GetTime<std::chrono::seconds>();
```

Comment:
```suggestion
            auto current_time{GetTime<std::chrono::seconds>()};
```

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/25079/commits/626b636b193baae81a697febd41a712f94a8a702. Thanks for suggestion.